### ISSUE TITLE
Tvorba startovky - draw tool neřeší nastavený start prvního závodníka

### ISSUE THREAD

**Issue by lukaskett** [fvacek/quickbox#740]
Když si v záložce kategorie krásně vyplním start prvního závodníka
![image](https://user-images.githubusercontent.com/29544822/171055947-78924934-a03e-4a3b-9973-817396a2d605.png)
, tak ho nástroj na rozvržení startu vůbec nebere v potaz a všechny kategorie začínají na 0.

Plus jsem měl pocit, že dřív šlo s bloky (po výběru) kategorií posouvat kolečkem ve směru času startu prvního závodníka, což mi teď nefungovalo. Musel jsem vždy ručně kliknout na začátek řádku a vybrat `Nastavit start v řádku`.
![image](https://user-images.githubusercontent.com/29544822/171056153-e805f6ba-4a59-4990-918a-6ac8d46acaa0.png)

**Comment by fvacek** [comment]
Ano, to je pravda, funguje to naopak, co se vyplni v draw-tool, to se pak zapise do tabulky.

**Comment by lukaskett** [comment]
> Plus jsem měl pocit, že dřív šlo s bloky (po výběru) kategorií posouvat kolečkem ve směru času startu prvního závodníka, což mi teď nefungovalo. Musel jsem vždy ručně kliknout na začátek řádku a vybrat Nastavit start v řádku.

Už jsem na to přišel, musím být kurzorem myši přesně na čísle s časem startu v řádku a pak funguje otáčení kolečkem na změnu hodnoty - zároveň se blok kategorie posouvá.

**Comment by fvacek** [comment]
koleckem to jde, pokud mas ukazatel mysi na cisle startu startovniho slotu

**Comment by lukaskett** [comment]
> Ano, to je pravda, funguje to naopak, co se vyplni v draw-tool, to se pak zapise do tabulky.

A šlo by to upravit na obousměrnou funkcionalitu? Tj aby se dle v tabulce vyplněných hodnot naposouval i layout.

**Comment by xkenia** [comment]
Zkusil jsem to upravit u sebe commitem [#3a350d5c5c01e047dbc38dcc3d67bf3537aecaf6](https://github.com/Quick-Event/quickbox/commit/3a350d5c5c01e047dbc38dcc3d67bf3537aecaf6) , ale chtelo by to poradne otestovat.

**Comment by arnost00** [comment]
Vzal jsem commit od @xkenia, zkusil ho, opravil a dal pull request.

**Comment by fvacek** [comment]
Bojim se, ze ta uprava nic neresi, nastvi cas prvni kategorie ve slotu podle tabulky, ale ty ostatni se za ni stejne naskladaji, takze jiste nebudou mit startovaci casy podle tabulky.
